tests/resources/quality-matrix-replication-source-request.json

```diff
@@ -45,7 +45,7 @@
     "replication_source": {
       "terms": {
         "field": "properties.ccm:replicationsource.keyword",
-        "size": 10000
```

Comment:
Oh, I guess those depend on the .env file used to run the tests.
I wonder how the pipeline worked before or after that change!?

Reply:
Nevermind, found the other related changes that make that work :)